Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -50,13 +50,21 @@ public final class SwiftTargetBuildDescription {
 
     /// Path to the bundle generated for this module (if any).
     var bundlePath: AbsolutePath? {
-        if let bundleName = target.underlyingTarget.potentialBundleName, !resources.isEmpty {
+        if let bundleName = target.underlyingTarget.potentialBundleName, needsResourceBundle {
             return self.buildParameters.bundlePath(named: bundleName)
         } else {
             return .none
         }
     }
 
+    private var needsResourceBundle: Bool {
+        return resources.filter { $0.rule != .embedInCode }.isEmpty == false
```

Comment:
`! needsResourceEmbedding`?

Reply:
My idea was to allow a mix of embedded and not embedded resources, so each of these is true if there's at least one file to handle.